corehq/apps/registration/utils.py

```diff
@@ -256,14 +259,32 @@ def send_new_request_update_email(user, requesting_ip, entity_name, entity_type=
         logging.warning("Can't send email, but the message was:\n%s" % message)
 
 
-def send_mobile_experience_reminder(recipient, full_name):
+def project_logo_emails_context(domain, couch_user=None):
+    if couch_user:
+        user_domains = getattr(couch_user, 'domains', None)
+        if user_domains:
+            domain = user_domains[0]
+    if domain and USE_LOGO_IN_SYSTEM_EMAILS.enabled(domain):
+        try:
+            image_reference = LogoForSystemEmailsReference.objects.get(domain=domain)
+            return {
+                "base_container_template": "registration/email/base_templates/_base_container_project_logo.html",
+                "link_to_logo": image_reference.full_url_to_image()
+            }
+        except LogoForSystemEmailsReference.DoesNotExist:
```

Comment:
would it be useful to log this in case something is missconfigured and has to be debugged?